src/extension/features/budget/pacing/index.js

```diff
@@ -18,6 +18,18 @@ export class Pacing extends Feature {
     return isCurrentRouteBudgetPage();
   }
 
+  observe(changedNodes) {
+    if (!this.shouldInvoke()) return;
+
+    if (
+      changedNodes.has(
+        'budget-table-row js-budget-table-row is-sub-category is-debt-payment-category'
```

Comment:
this seems like a weird node to react to if we're trying to react to "any row is added" - is there any chance the `budget-table` node updates? 

honestly, if you're up for it, it would be even better to migrate this feature into using the `addToolkitEmberHook` on the `budget-table` component.

Reply:
I'll migrate this. Thanks for the suggestion!